game.exs

```diff
@@ -1,7 +1,39 @@
 defmodule GuessingGame do
 
-  def hi() do
-    IO.puts("hello world")
+  def compare(n, t) do
+    cond do
+      n < t -> :low
+      n == t -> :equal
+      n > t -> :high
+    end
+  end
+
+  def print_state(:low),      do: IO.puts("Too low")
+  def print_state(:equal),    do: IO.puts("That's the right number, you win!")
```

Comment:
I'd actually keep `case` for `print_state`, I just wanted to point out that you had options in my last review :)